tests/phpunit/integration/Core/Conversion_Tracking/Conversion_TrackingTest.php

```diff
@@ -41,6 +59,12 @@ public function tear_down() {
 		parent::tear_down();
 
 		Conversion_Tracking::$providers = array();
+
+		$original_conversion_tracking_settings = array(
+			'enabled' => true,
+		);
+
+		$this->conversion_tracking_settings->set( $original_conversion_tracking_settings );
```

Comment:
This shouldn't be needed since changes to options shouldn't persist between tests as WP will rollback the transaction. Feel free to double check to make sure though 👍